docs/content/getting-started.md

```diff
@@ -139,10 +139,10 @@ Returning to `graph/schema.resolvers.go`, let's implement the bodies of those au
 
 ```go
 func (r *mutationResolver) CreateTodo(ctx context.Context, input model.NewTodo) (*model.Todo, error) {
-	rand, _ := rand.Int(rand.Reader, big.NewInt(100))
+	randNumber, _ := rand.Int(rand.Reader, big.NewInt(100))
```

Comment:
rename is variable because var `rand` collides with imported package name `rand`